cli/pom.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <parent>
+    <artifactId>alizer</artifactId>
+    <groupId>com.redhat.devtools.alizer</groupId>
+    <version>0.1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>cli</artifactId>
+  <properties>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
```

Comment:
Not sure the non quarkus properties are required as they are probably herited from jboss-parent, to be checked